recipes/foreman.rb

```diff
@@ -0,0 +1,11 @@
+REPO = "https://raw.githubusercontent.com/spartansystems/spartan-composer-recipes/master/files/".freeze
+
+stage_one do
+  copy_from_repo "Procfile", repo: REPO
+end
+__END__
+
+name: foreman
+description: "Add the default Foreman Procfile"
```

Comment:
Any recommendation on a new name? Procfile? It is for Foreman. Just not local Foreman lol.